pkg/api/ocpbuilddata/types.go

```diff
@@ -85,14 +85,15 @@ type OCPImageConfigFrom struct {
 type OCPImageConfigFromStream struct {
 	Stream string `json:"stream"`
 	Member string `json:"member"`
+	Image  string `json:"image"`
 }
 
 func (icfs OCPImageConfigFromStream) validate() error {
-	if icfs.Stream == "" && icfs.Member == "" {
-		return errors.New("both stream and member were unset")
+	if icfs.Stream == "" && icfs.Member == "" && icfs.Image == "" {
+		return errors.New("all stream, member and image were unset")
 	}
-	if icfs.Stream != "" && icfs.Member != "" {
-		return fmt.Errorf("both stream(%s) and member(%s) were set", icfs.Stream, icfs.Member)
+	if icfs.Stream != "" && icfs.Member != "" && icfs.Image != "" {
```

Comment:
This means it is okay to have 2 of the 3 set. Are we sure that it isn't necessary to validate that exactly 1 is set?